CHANGELOG.md

```diff
@@ -672,6 +676,15 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Fixed passing `_ddp_params_and_buffers_to_ignore` ([#11949](https://github.com/PyTorchLightning/pytorch-lightning/pull/11949))
 
 
+- Fixed an `AttributeError` when calling `save_hyperparameters` and no parameters need saving ([#11827](https://github.com/PyTorchLightning/pytorch-lightning/pull/11827))
```

Comment:
the other entries were incorrectly moved to the already released section somewhere in other PRs. Verified that they are not present on release branch, so updating them here.